# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1350489095 14400
#      Wed Oct 17 11:51:35 2012 -0400
# Node ID 94d21131fefd1eb164f225f33b676eb3cd6247cc
# Parent  360adb4a3136abfe86b1224e01ac54f8d6685e18
# Parent  d23eed1c5d5d03ed54bb2bf8bc2695c7e6632296
maint: periodic merge of stable to default

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -63,8 +63,9 @@ 95c43fc8dbe1a07a46fefb3372df5b2309d874fd
 0000000000000000000000000000000000000000 release-3-2-4
 e320928eeb3aa2370b792e83dafc3e0ddecdc871 release-3-2-4
 ba4d6343524b406b0d15aee34579f80783581c54 release-3-6-1
 704f7895eef03008dd79848eb9da4bfb40787d73 release-3-6-0
 f947d2922febf12dcd1fb6e21b356756ecb54e55 rc-3-6-2-0
 4460c4fb20e6a5d3b1972fa737d4e00eb921545a rc-3-6-2-2
 551566201318bf615b27c60ccf9368f4844008bd release-3-6-2
 a95432e7309ca6fc776c02939264bb6d443f3525 release-3-6-3
+2e8eb9ac43a5f8cfaf0423814a312ed47cb80485 rc-3-6-4-0
diff --git a/scripts/sparse/svds.m b/scripts/sparse/svds.m
--- a/scripts/sparse/svds.m
+++ b/scripts/sparse/svds.m
@@ -283,15 +283,15 @@ endfunction
 %!testif HAVE_ARPACK
 %! [u2,s2,v2,flag] = svds (zeros (10), k);
 %! assert (u2, eye (10, k));
 %! assert (s2, zeros (k));
 %! assert (v2, eye (10, 7));
 %!
 %!testif HAVE_ARPACK
 %! s = svds (speye (10));
-%! assert (s, ones (6, 1), 2*eps);
+%! assert (s, ones (6, 1), 4*eps);
 
 %!test
 %! ## Restore random number generator seeds at end of tests
 %! rand ("state", rand_state);
 %! randn ("state", randn_state);
 
